src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt

```diff
@@ -12,7 +12,7 @@ class PrivacyModule(
 ) : Module {
     private val patterns: List<Regex> = listOf(
         """\(Session ID is token:""".toRegex(),
-        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}\b(?!])""".toRegex(),
+        """[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}""".toRegex(),
```

Comment:
Only removing the ending part `\b(?!])` should be able to fix this issue while still keeping the functionality of not hiding user mentions containing email addresses.

Reply:
Mentioning doesn't really do anything though, and once the user name has been changed from the email, then the mention will instead only display the raw email because that user name doesn't exist anymore, and arisa won't mark it as private

Reply:
Yeah, nevermind. You're probably right. Changed to just removing `\b(?!])`